apps/ledger-live-mobile/src/screens/SendFunds/01c-SelectNft.tsx

```diff
@@ -26,36 +18,56 @@ import { SendFundsNavigatorStackParamList } from "../../components/RootNavigator
 const MAX_NFTS_FIRST_RENDER = 8;
 const NFTS_TO_ADD_ON_LIST_END_REACHED = 8;
 
-const NftRow = memo(({ account, nft }: { account: Account | TokenAccount; nft: ProtoNFT }) => {
+const NftRow = memo(({ account, nft }: { account: Account; nft: ProtoNFT }) => {
   const { t } = useTranslation();
   const navigation =
     useNavigation<StackNavigatorNavigation<SendFundsNavigatorStackParamList, ScreenName.SendNft>>();
   const { colors } = useTheme();
-  const { status, metadata } = useNftMetadata(nft?.contract, nft?.tokenId, nft?.currencyId) as {
-    status: NFTResource["status"];
-    metadata?: NFTMetadataResponse["result"] & NFTCollectionMetadataResponse["result"];
-  };
+  const { status, metadata } = useNftMetadata(nft?.contract, nft?.tokenId, nft?.currencyId);
 
   const nftCapabilities = useMemo(() => getNftCapabilities(nft), [nft]);
 
   const goToRecipientSelection = useCallback(() => {
     const bridge = getAccountBridge(account);
 
-    let transaction = bridge.createTransaction(account);
-    transaction = bridge.updateTransaction(transaction, {
-      tokenIds: [nft?.tokenId],
-      // Quantity is set to null first to allow the user to change it on the amount page
-      quantities: [nftCapabilities.hasQuantity ? null : new BigNumber(1)],
-      collection: nft?.contract,
-      mode: `${nft?.standard?.toLowerCase()}.transfer`,
-    });
+    const defaultTransaction = bridge.createTransaction(account);
+    let transaction;
+    if (account.currency.family === "evm") {
+      transaction = bridge.updateTransaction(defaultTransaction, {
+        nft: {
+          contract: nft?.contract,
+          tokenId: nft?.tokenId,
+          // When available, quantity is set to Infinity (considered null) first for
+          // the next UI. It allows the user to not have a default value during
+          // the first screen, like 0 or 1, which could trigger an error
```

Comment:
If I understood well, we try to bypass the transaction status thing ? Isn't there a way to set a better defaultValue ? Why is 1 wrong ?

Reply:
No, the transaction status will still contain an error because the user can't send an Infinity of NFT.
The goal here is to not have a type `quantity: BigNumber | null` because that would mean testing if the property is a `BigNumber` everytime you want to use it, which is a poor DX imo.
And the reason why we would need a `null` in the first place is to avoid having an already set value when reaching the Amount screen. Product wanted to have a blank value that the user would have to set manually rather than a `1` default value. And with the way the bridge is connected to the transaction which is itself connected to the input as dependencies, this is the best solution I found to keep a coherent typing/DX while maintaining the UX.